main.go

```diff
@@ -170,7 +170,7 @@ func main() {
 		MTLS *struct {
 			RawCA  []byte `json:"ca"`
 			CAPath string `json:"caPath"`
-			ca     *x509.Certificate
+			ca     []*x509.Certificate
```

Comment:
Same here

Reply:
Also, i think the CAPath field should be an array to match

Reply:
Once we change the CAPath field to and array called CAPaths, we will need to adjust the integration test configuration here as well: https://github.com/observatorium/api/blob/master/test/config/tenants.yaml#L26

Reply:
@squat 
I think CAPath should still a single file path, but not multiple?
To support multiple ca certfiicates, the common usage pattern is to put multiple certificates into one ca file, but not provide multiple ca files.

Reply:
Aha ok that makes a lot of sense. Sgtm